index.bs

```diff
@@ -1877,6 +1877,53 @@ If the bytes in the buffer have a natural intepretation
 as one of the other TypedArray types, provide that instead.
 For example, if the bytes represent Float32 values, use a {{Float32Array}}.
 
+<h3 id="allow-microtask-switches">A Promise represents a value, not a callback</h3>
```

Comment:
Agreed with Jeffrey.

Reply:
Makes sense. 

> Maybe a title like "Promise resolution should not expire", although that could probably be improved.

Resolution does often expire though. The principle here is to avoid synchronous expiry as a design (a pattern with some callbacks or even event listeners).

How about "A Promise represents completion or a value, not a callback" ?